bochs/iodev/usb/Makefile.in

```diff
@@ -50,7 +50,7 @@ top_builddir    = ../..
 LIBTOOL=@LIBTOOL@
 WIN32_DLL_IMPORT_LIBRARY=../../@WIN32_DLL_IMPORT_LIB@
 
-USBDEV_OBJS = usb_hid.o usb_hub.o usb_msd.o usb_printer.o usb_floppy.o
+USBDEV_OBJS = usb_hid.o usb_hub.o usb_msd.o usb_uasp.o usb_printer.o usb_floppy.o
```

Comment:
I suggest to follow some another file like usd_msd and see where it present
I see

USBDEV_DLL_TARGETS = bx_usb_hid.dll bx_usb_hub.dll bx_usb_msd.dll bx_usb_printer.dll bx_usb_floppy.dll

should you add there ?

Also please add dep like below:

usb_msd.o: usb_msd.@CPP_SUFFIX@ ../iodev.h ../../bochs.h ../../config.h \
 ../../osdep.h ../../gui/paramtree.h ../../logio.h \
 ../../instrument/stubs/instrument.h ../../misc/bswap.h ../../plugin.h \
 ../../extplugin.h ../../param_names.h ../../pc_system.h \
 ../../bx_debug/debug.h ../../config.h ../../osdep.h \
 ../../memory/memory-bochs.h ../../gui/siminterface.h ../../gui/gui.h \
 usb_common.h usb_pcap.h ../hdimage/cdrom.h ../hdimage/hdimage.h \
 scsi_device.h usb_msd.h

Reply:
I believe I have made the correct additions you have asked for.